Fix smoke tests for 6.4

Currently , smoke tests are failing for 6.4, This issue will cover fining the CLI and API tests as UI tests needs refactoring by using airgun

tests.foreman.endtoend.test_api_endtoend.AvailableURLsTestCase.test_positive_get_links

```
        )
E       AssertionError: Items in the first set but not the second:
E       'ansible_roles'
E       'auth_sources'
E       'upstream_subscriptions'
E       'puppet_hosts'
E       'content_credentials'
E       'http_proxies'
E       'debs'
E       'table_preferences'
E       'auth_source_internals'
E       'auth_source_externals'
E       'docker_manifest_lists'
E       'personal_access_tokens'
```

tests.foreman.endtoend.test_cli_endtoend.EndToEndTestCase.test_positive_end_to_end
```
        self.assertEqual(
            content_host['content-information']['content-view'],
>           content_view['name']
        )
E       AssertionError: {'id': '232', 'name': 'oPypdTC2uo'} != 'oPypdTC2uo'

tests/foreman/endtoend/test_cli_endtoend.py:375: AssertionError
```

--- a/tests/foreman/endtoend/test_cli_endtoend.py
+++ b/tests/foreman/endtoend/test_cli_endtoend.py
@@ -371,12 +371,14 @@ class EndToEndTestCase(CLITestCase, ClientProvisioningMixin):
         ).info({'id': content_host['id']})
         # check that content view matches what we passed
         self.assertEqual(
-            content_host['content-information']['content-view'],
+            content_host['content-information'][
+                'content-view']['name'],
             content_view['name']
         )
         # check that lifecycle environment matches
         self.assertEqual(
-            content_host['content-information']['lifecycle-environment'],
+            content_host['content-information'][
+                'lifecycle-environment']['name'],
             lifecycle_environment['name']
         )
 

